common/util.py

```diff
@@ -0,0 +1,16 @@
+import countrynames
+from zavod import Zavod
+
+
+def make_oc_company_id(context: Zavod, jurisdiction: str, company_nr: str) -> str:
+    """
+    generate company id in open corporates format
+    """
+    # ensure country
+    cc = countrynames.to_code(jurisdiction)
```

Comment:
I'm pretty sure the OpenCorporates and FollowTheMoney definiton of countries doesn't fully match (see the US state URLs), so I'd avoid this. Also doesn't need `context` if there's no country normalisation.